feature-utils/poly-look/src/react-components/icons/circle-xmark.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export function CircleXMark({ fill, ...rest }) {
+  return (
+    <svg
+      data-testid="circle-xmark"
+      {...rest}
+      width="24"
```

Comment:
This is essentially the same as above, except for the path and the test-id. Maybe DRY them to a single component? There might be more in the future...

Reply:
I agree :)

Reply:
Right now there are only 3 so it would make sense, but in the future we'll have all icons like this. That means it's going to be a pretty long file, so I'm more tempted to leave it as it is.